### PR TITLE
Some doc fixes/improvements

### PR DESCRIPTION
**Pull request by bvdvecht**

### CODE REVIEW

File: docs/quickstart/using-sdk.rst [hunk 1]
```diff
@@ -226,6 +228,13 @@ The code for ``bob`` will be very similar, with the only difference being that `
 Running this application files using ``netqasm simulate`` prints the outcomes of the two nodes.
 Since by default no noise is used, their outcomes will always be equal.
 
+
+.. note::
+
+   You may see a warning coming from NetSquid saying ``WARNING: a [sic] expression handler missing``.
```
Comment (QFer) [inline docs/quickstart/using-sdk.rst]: Maybe add a separate block or paragraph with 'known issues'. I know from experience that I check such paragraph for solved issues once in a while. With these notes throughout the text it is more difficult to maintain them when there are problems solved.

File: docs/quickstart/using-sdk.rst [hunk 2]
```diff
@@ -541,11 +550,11 @@ What happens under the hood, is that these if-statements are compiled into branc
 
 
 .. tip::
-   Checkout the documentation for :class:`~.Future`.
+   Check aout the documentation for :class:`~.Future`.
```
Comment (QFer) [inline docs/quickstart/using-sdk.rst]: Check out

File: docs/quickstart/using-sdk.rst [hunk 3]
```diff
@@ -45,8 +45,7 @@ Open the file ``app_alice.py`` in your favorite editor/IDE and add the following
    load different classes depending on its value. What this allows us to do is to simulate an
    application on different simulators, without changing anything, even imports.
 
-What this code does is to setup a connection to the underlying (simulated) ``QNodeOS`` which can handle the NetQASM-instructions.
-More information about ``QNodeOS`` can be found :qnodeos:`here <>`.
+What this code does is to setup a connection to the underlying (simulated) quantum node controller called QNodeOS, which can handle the NetQASM-instructions.
```
Comment (QFer) [inline docs/quickstart/using-sdk.rst]: QNodeOS -> ``QNodeOS``

**Review by QFer — CHANGES_REQUESTED** [review]

**Comment by bvdvecht** [comment]
@QFer Fixed the spelling mistakes + added a separate Known Issues section

**Review by QFer — APPROVED** [review]

**Pull request merged**